tools/filter_variant_SnpSift_cmd/filter_variant_SnpSift_cmd/.shed.yml

```diff
@@ -0,0 +1,15 @@
+# .shed.yml supporting automatic pushes.
+owner: artbio
+name: filer_variant_SnpSift_cmd
+description: filer_variant_SnpSift_cmd creates the filter line parameter to filter variants in SnpSift tool
+long_description: |
+    Takes a description table and a VCF file to create the command line for filtering in SnpSift tool
```

Comment:
a description table of what ? c'est difficile de se figurer l'usage de l'outil. Si tu veux qu'il soit utilisé, il faut rendre cela plus évident